app/models/patient_import_row.rb

```diff
@@ -8,6 +8,11 @@ class PatientImportRow
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :nhs_number, length: { is: 10 }, allow_blank: true
+  validates :gender_code,
+            inclusion: {
+              in: Patient.gender_codes.keys
+            },
+            allow_blank: true
```

Comment:
I think we don't want to allow blank here, since we've got a default in the `def gender_code` method anyway it'll always have a value.

Reply:
Agreed, I've removed the `allow_blank`.